anaconda_opentelemetry/exporters.py

```diff
@@ -0,0 +1,35 @@
+from opentelemetry.exporter.otlp.proto.http.metric_exporter import OTLPMetricExporter as OTLPMetricExporterHTTP
```

Comment:
If you need to make a new subclass, for every class we could use(tracing-http, tracing-grpc,metrics-http, metrics-grpc, logging-http, logging-grpc), is this any better than a real shim per type? I usually prefer composition over inheritance. Just a question. :)